ePBFT/src/pbft/libbyz/test/test_ledger_replay.cpp

```diff
@@ -67,11 +68,18 @@ NodeInfo get_node_info()
   return node_info;
 }
 
-void init_replica(std::vector<char>& service_mem)
+void create_replica(
+  std::vector<char>& service_mem,
```

Comment:
`std::vector<uint8_t>`?

Reply:
Replica's constructor expects `char`. It could probably be made `uint8_t` but it is out of the scope of this PR.